subcommand/control-plane/command_test.go

```diff
@@ -1026,7 +1081,7 @@ func getExpectedDataplaneCfgJSON() string {
 	"consul": {
 	  "addresses": "127.0.0.1",
 	  "grpcPort": %d,
-	  "serverWatchDisabled": false
+	  "serverWatchDisabled": false%s
```

Comment:
Is this a typo?

```suggestion
	  "serverWatchDisabled": false
```

Reply:
No. This was added to accommodate the `credentials` field when consul login is enabled. ([ref](https://github.com/hashicorp/consul-ecs/blob/1903328205d099c42d48e940fbeee49430db8f26/subcommand/control-plane/command_test.go#L941))

Reply:
:+1: Got it, thanks.